### PR TITLE
1.Modify the layout and some contents;2.Change the license img source to the url.

### PR DESCRIPTION
**Pull request by Tenktau**

### CODE REVIEW

**Comment by Tenktau** [comment]
Deploy失败我在自己的仓库也遇到过，原因应该是Git Action的Bot没有仓库的Write权限。

File: docs/ch3/reference.md
Comment (wsong83) [inline docs/ch3/reference.md:1]: I dont think it is a good idea to remove stuff that might be useful in the future? I would like this file to be restored before accepting the PR.

**Comment by wsong83** [comment]
check失败不是什么问题。但是问题是不要轻易删除暂时看起来没用的内容。关于文件结构的修改暂时先这样吧。

**Comment by wsong83** [comment]
The original docs/ch3/reference.md was not empty?

**Comment by wsong83** [comment]
Thanks!

**Pull request merged**